### ISSUE TITLE
[Accessibility] Side docs

### ISSUE THREAD

**Issue by microsoftsam** [Microsoft/pxt#2421]
Tabbing in the side docs is not accessible

**Comment by veler** [comment]
I need more precisions. At a moment the user was trapped in the side docs but this solution has been canceled. Which UX do you expect?
